### DISCUSSION TITLE
Browser Path

### DISCUSSION THREAD

**Discussion by iamtaufik** [domcloud/domcloud#49] [General]
Where the browser path is located?

**Comment by willnode** [comment]
Do you mean the domain URL?

**Reply by iamtaufik** [reply]
nope, I made a whatsapp bot using Puppeteer lib and needed the location of the browser used. for example the location of the browser in `/usr/bin/google-chrome-stable`

**Reply by willnode** [reply]
Doesn't Puppeteer download its own Chrome browser? Located in `$HOME/.cache/puppeteer` according to [the doc](https://pptr.dev/#:~:text=When%20you%20install%20Puppeteer%2C%20it%20automatically%20downloads%20a%20recent%20version%20of%20Chrome%20for%20Testing%20(~170MB%20macOS%2C%20~282MB%20Linux%2C%20~280MB%20Windows)%20that%20is%20guaranteed%20to%20work%20with%20Puppeteer.%20The%20browser%20is%20downloaded%20to%20the%20%24HOME/.cache/puppeteer%20folder%20by%20default%20(starting%20with%20Puppeteer%20v19.0.0).)
